openedx/core/lib/api/tests/test_paginators.py

```diff
@@ -107,8 +107,7 @@ def test_paginated_results_last_keyword(self):
         id_range = get_object_range(page_number, page_size)
         db_objects = [build_mock_object(obj_id) for obj_id in id_range]
         self.mock_model.objects.filter = MagicMock(return_value=db_objects)
-
-        page = paginate_search_results(self.mock_model, self.search_results, self.default_size, 'last')
+        page = paginate_search_results(self.mock_model, self.search_results, page_size, 'last')
```

Comment:
Any particular reason that we are using `page_size` instead of `self.default_size` here